pkg/job_controller/job.go

```diff
@@ -197,7 +199,14 @@ func (jc *JobController) ReconcileJobs(
 				jobStatus.ReplicaStatuses[rtype].Succeeded += jobStatus.ReplicaStatuses[rtype].Active
 				jobStatus.ReplicaStatuses[rtype].Active = 0
 			}
+
+			// job finished, create the model version
+			err = jc.createModelVersion(metaObject, modelVersion, pods, &jobStatus)
```

Comment:
step into this function when `modelVersion` is not nil, otherwise it will be dereferenced and panic